tests/foreman/cli/test_computeresource.py

```diff
@@ -291,3 +291,53 @@ def test_update_negative(self, options):
         options.pop('new-name', None)
         for key in options.keys():
             self.assertEqual(comp_res[key], result.stdout[key])
+
+    @data('true', 'false')
+    def test_set_console_password_v1(self, set_console_password):
+        """@Test: Create a compute resource with ``--set-console-password``.
+
+        @Feature: Compute Resource
+
+        @Assert: The option is accepted, and the passed-in value can be read
+        back.
+
+        Targets BZ 1100344.
+
+        """
+        name = gen_string('utf8')
+        result = ComputeResource.create({
+            'name': name,
+            'provider': 'Libvirt',
+            'set-console-password': set_console_password,
+            'url': gen_url(),
+        })
+        self.assertEqual(result.return_code, 0)
+        self.assertEqual(len(result.stderr), 0)
+
+    @data('true', 'false')
+    def test_set_console_password_v2(self, set_console_password):
+        """@Test: Update a compute resource with ``--set-console-password``.
+
+        @Feature: Compute Resource
+
+        @Assert: The option is accepted, and the passed-in value can be read
```

Comment:
Same as before, unless ``"the passed-in value"`` is not the ``console password`` but the entity itself?